spec/controllers/api/v1/courses_controller_spec.rb

```diff
@@ -133,7 +96,7 @@ def format_timestamp(timestamp)
       describe 'returned courses in JSON' do
         subject { response.body }
 
-        it { should_not have_courses }
+        it { should have_courses([]) }
```

Comment:
me no likey

Reply:
should be_empty ?

Reply:
Why was `should_not have_courses` removed?

Reply:
Have rewrote the logic so should_not works 👍

Reply:
:rofl: should_not work :roll_eyes: